pkg/daemon/daemon.go

```diff
@@ -203,6 +205,13 @@ const (
 	// Where nmstate writes the link files if it persisted ifnames.
 	// https://github.com/nmstate/nmstate/blob/03c7b03bd4c9b0067d3811dbbf72635201519356/rust/src/cli/persist_nic.rs#L32-L36
 	systemdNetworkDir = "etc/systemd/network"
+
+	// prefetchTimeout is the timeout for the prefetch process.
+	prefetchTimeout = 10 * time.Minute
+
+	// minFreeStorageAfterPrefetch is the minimum amount of storage in bytes available on the root filesystem
+	// after prefetching images.
+	minFreeStorageAfterPrefetch int64 = 32 * 1024 * 1024 * 1024 // 32GB
```

Comment:
thoughts?

Reply:
Not sure if we already collect some metrics around available free spaces for cluster for which it is targeted. If it exist, that will help to guess this value better. Free space is relative based on what kind of application is running on a cluster. A storage hungry application can run into no space sooner.